spec/services/api_login_manager_spec.rb

```diff
@@ -1,87 +1,288 @@
       end
 
       subject do
         described_class.new(email: user.email, password: password).call
       end
 
       it "update user's auth_token" do
-        skip 'Una vez que definan a user, este test va a fallar. ¿Por qué?'
-        expect { subject }.to change { user.auth_token }
+        expect { subject }.to change { user.reload.auth_token }
       end
 
       it 'returns the auth_token' do
-        skip 'Implementar'
+        expect(subject).to eq user.reload.auth_token
       end
     end
 
     context 'when no email is provided' do
-      subject {  } # implementar
+      subject { described_class.new(password: password) }
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns EMPTY_EMAIL error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to eq 'El email no puede estar en blanco'
       end
     end
 
     context 'when no password is provided' do
-      subject {  } # implementar
+      subject { described_class.new(email: user.email) }
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns EMPTY_PASSWORD error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to eq 'La contraseña no puede estar en blanco'
       end
     end
 
     context 'when the email is incorrect' do
-      subject {  } # implementar
+      subject do
+        described_class.new(email: 'wrong@email.com', password: password)
+      end
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns USER_NOT_FOUND error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to eq 'El usuario no existe'
       end
     end
 
     context 'when the password is incorrect' do
-      subject {  } # implementar
+      subject do
+        described_class.new(email: user.email, password: 'wrong_password')
+      end
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns WRONG_PASSWORD error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to eq 'La contraseña es incorrecta'
+      end
+    end
+
+    context 'when the external validation fail' do
+      before do
+        expect(ExternalValidator).to receive(:call).and_return(false)
+      end
+
+      subject do
+        described_class.new(email: user.email, password: password)
+      end
+
+      it 'returns false' do
+        expect(subject.call).to be_falsey
+      end
+
+      it 'returns WRONG_PASSWORD error with a reader' do
+        subject.call
+        expect(subject.error).to eq 'El usuario ya no tiene cuota disponible'
+      end
+    end
+  end
+end
```

Comment:
En todos los `be_falsey`, siendo que el mismo test dice `it 'returns false'`, serían más específicos con un `be false`.